app/core/components/ranker/default.py

```diff
@@ -194,6 +194,12 @@ def _score_obs(self, program: Program, obs: Observation) -> Scores:
                                               np.ones(1, dtype=int) * program.band,
                                               np.ones(1) * 0.8,
                                               program.thesis)
+        print(f'{obs.id:20}  exec: {obs.exec_time().total_seconds() / 3600.:.2f} '\
```

Comment:
This prints should either be in `logging` or removed as is not a good practice to have direct prints in the code.